dev/com.ibm.ws.security.javaeesec/resources/com/ibm/ws/security/javaeesec/internal/resources/JavaEESecMessages.nlsprops

```diff
@@ -27,14 +27,14 @@ JAVAEESEC_ERROR_NO_VALIDATION=CWWKS1911E: No IdentityStore object supports the u
 JAVAEESEC_ERROR_NO_HAM.explanation=HttpAuthenticationMechanism object cannot be identified for authentication. It could be there is no object, or multiple objects which corresponds the specified condition. Make sure that there is one HttpAuthenticationMechanism in the application.
 JAVAEESEC_ERROR_NO_HAM.useraction=Make sure that there is one HttpAuthenticationMechanism in the application.
 
-JAVAEESEC_ERROR_NO_HAM_PROPS=CWWKS1913E: HAMProperties object cannot be identified for authentication. Most likely, there is not any HttpAuthenticationMechanism object in the application. Make sure that one HttpAuthenticationMechanism objct is defined.
-JAVAEESEC_ERROR_NO_HAM_PROPS.explanation=HAMProperties object cannot be identified for authentication. It could be there is not any HttpAuthenticationMechanism object defined in the application.
-JAVAEESEC_ERROR_NO_HAM_PROPS.useraction=Make sure that there is one HttpAuthenticationMechanism in the application.
+JAVAEESEC_ERROR_NO_MODULE_PROPS=CWWKS1913E: The ModuleProperties object for authenticating the {0} application cannot be found, likely because the application does not contain a HttpAuthenticationMechanism object. Make sure that one HttpAuthenticationMechanism object is defined for each module.
+JAVAEESEC_ERROR_NO_MODULE_PROPS.explanation=The ModuleProperties object, which is required to authenticate the application, cannot be identified. This error typically results from a HttpAuthenticationMechanism object not being defined in the application.
+JAVAEESEC_ERROR_NO_MODULE_PROPS.useraction=Make sure that each module in the application contains a HttpAuthenticationMechanism object.
 
-JAVAEESEC_WARNING_NO_REALM_NAME=CWWKS1914W: The realm name for the basic authentication is not set, the value defaultRealm is being used.
-JAVAEESEC_WARNING_NO_REALM_NAME.explanation=The realm name for the basic authentication is not set, the default value defaultRealm is being used.
-JAVAEESEC_WARNING_NO_REALM_NAME.useraction=If the realm name needs to be set, please set it by using BasicAuthenticationMechanismDefinition annotation in the application.
+JAVAEESEC_ERROR_MULTIPLE_HTTPAUTHMECHS=CWWKS1914E: The login configuration for the module {0} in the application {1} because multiple HttpAuthenticationMechanism implementation are found. The list of classes is {2}. Most likely, this is an application packaging issue. Please make sure that there is only one HttpAuthenticationMechanism implementation in the module.
+JAVAEESEC_ERROR_MULTIPLE_HTTPAUTHMECHS.explanation=The login configuration for the module failed because more than one HttpAuthenticationMechanism implementation are enabled. This error is typically caused by an application packaging issue.
+JAVAEESEC_ERROR_MULTIPLE_HTTPAUTHMECHS.useraction=The login configuration for the module failed because more than one HttpAuthenticationMechanism implementation are enabled. This error is typically caused by an application packaging issue.
```

Comment:
What action should users take here? Looks identical to the explanation - copy/paste issue? In my review it read "Make sure that each module in the application contains only one HttpAuthenticationMechanism implementation."

Reply:
Modified as follows:
JAVAEESEC_ERROR_MULTIPLE_HTTPAUTHMECHS.useraction=Make sure that there is only one HttpAuthenticationMechanism implementation in the module.